### DISCUSSION TITLE
How to activate the dark mode on windows?

### DISCUSSION THREAD

**Discussion by arthackrc** [transmission/transmission#3722] [Support] [answered]
I've used the latest nightly build from the site. But it doesn't have a setting to enable the dark mode on my windows 11. Is there a dark mode or not?

**Comment by schobernoise** [comment]
Would like to know that too, bump!

**Comment by Pentaphon** [comment, marked as answer]
There's no dark mode on Transmission itself.

**Reply by ngoclong19** [reply]
You can enable dark mode for Qt application. Please see https://github.com/transmission/transmission/issues/920.

**Reply by Pentaphon** [reply]
> You can enable dark mode for Qt application. Please see #920.

It's not an official dark mode, though. It's a workaround for a Qt bug that doesn't quite look right. Not sure why the devs do not leave an open issue so people can see that they intend to eventually add a proper dark mode once the Qt bug gets fixed.

![dark](https://user-images.githubusercontent.com/49374058/211267690-3d964be7-2ff7-464e-ba2a-1973b4d6f8df.png)
